packages/framework/src/Form/Constraints/ContainsValidator.php

```diff
@@ -17,14 +17,16 @@ public function validate($value, Constraint $constraint)
             throw new \Symfony\Component\Validator\Exception\UnexpectedTypeException($constraint, Contains::class);
         }
 
-        if (mb_strpos($value, $constraint->needle) === false) {
-            $this->context->addViolation(
-                $constraint->message,
-                [
-                    '{{ value }}' => $this->formatValue($value),
-                    '{{ needle }}' => $this->formatValue($constraint->needle),
-                ]
-            );
+        if (mb_strpos($value, $constraint->needle) !== false) {
+            return;
         }
+
+        $this->context->addViolation(
+            $constraint->message,
+            [
+                '{{ value }}' => $this->formatValue($value),
+                '{{ needle }}' => $this->formatValue($constraint->needle),
+            ]
+        );
```

Comment:
This could be solved by ignoring it via `ignoreTrailingIfWithOneInstruction`.
see https://github.com/slevomat/coding-standard/#slevomatcodingstandardcontrolstructuresearlyexit-

What do you think?

Reply:
That looks good

Reply:
> This could be solved by ignoring it via ignoreTrailingIfWithOneInstruction.
see https://github.com/slevomat/coding-standard/#slevomatcodingstandardcontrolstructuresearlyexit-

I have reverted some usages of early exits based on the mentioned rule in 7b8120ed267290f3a10c9f7b90456eb8cd9376ae

Reply:
nice